src/SFA.DAS.RoatpAssessor/SFA.DAS.RoatpAssessor.Web/Services/AssessorOverviewOrchestrator.cs

```diff
@@ -63,7 +127,11 @@ private static bool IsReadyForModeration(AssessorApplicationViewModel viewmodel)
             {
                 foreach (var section in sequence.Sections)
                 {
-                    if (section.Status == null || (!section.Status.Equals(SectionReviewStatus.Pass) && !section.Status.Equals(SectionReviewStatus.Fail) && !section.Status.Equals(SectionReviewStatus.NotRequired)))
+                    // TODO: Rework the logic according to requirements. Attention about AssessorSectionStatus.FailOutOf
+                    if (section.Status == null || (!section.Status.Equals(AssessorSectionStatus.Pass) && 
+                                                   !section.Status.Equals(AssessorSectionStatus.Fail) && 
+                                                   !section.Status.Equals(AssessorSectionStatus.NotRequired) &&
+                                                   !section.Status.Contains("OUT", StringComparison.InvariantCultureIgnoreCase)))
```

Comment:
This looks like a code smell, I'd convert these to a more explicit comparison).  What happens if we get a future status that has the letters 'out' in there?

Reply:
"{0} FAIL OUT OF {1}" is dynamic, which can be "2 FAIL OUT OF 5", "1FAIL OUT OF 3" or .... It cannot have explicit comparison. If you can suggest any other approach, I will be happy to implement it.

Reply:
Hopefully I've resolved this 👍